cumulus_library/enums.py

```diff
@@ -0,0 +1,17 @@
+""" Holds enums used across more than one module """
+from enum import Enum
+
+
+class ProtectedTableKeywords(Enum):
```

Comment:
What I'm trying to avoid with this is 'what if the string value changes for some reason?', which I've hit before - it's nice to have a layer between 'what the thing is in code' and 'what the actual string that gets written out to a database/other storage is' - this has been very helpful in the aggregator. But - i'm not wedded to it.